web/thesauruses/c/17/strings.json

```diff
@@ -0,0 +1,302 @@
+      ],
+      "name": "Is string empty?"
+    },
+    "concatenate_two_strings": {
+      "code": [
+        "char str1[] = \"hello\";",
+        "char str2[] = \" world\"",
+        "strcat(str1,str2);",
+        "puts(str1);"
+      ],
+      "name": "Concatenate/join two strings"
+    },
+    "concatenate_many_strings": {
+      "not-implemented": "true",
+      "name": "Concatenate/join many strings"
+    },
+    "is_all_alphabetical": {
+      "not-implemented":"true",
+      "name": "Is string all alphabetical characters?"
+    },
+    "is_all_numerical": {
+      "not-implemented":"true",
+      "name": "Is string all numerical characters?"
+    },
+    "is_all_alphanumeric": {
+      "not-implemented":"true",
+      "name": "Is string all alphanumeric characters?"
+    },
+    "is_decimal": {
+      "not-implemented":"true",
+      "name": "Is string a decimal number?"
+    },
+    "is_all_whitespaces": {
+      "not-implemented":"true",
+      "name": "Is string all whitespace characters?"
+    },
+    "is_all_uppercase": {
+      "not-implemented":"true",
+      "name": "Is string all uppercase characters?"
+    },
+    "is_all_lowercase": {
+      "not-implemented":"true",
+      "name": "Is string all lowercase characters?"
+    },
+    "is_in_titlecase": {
+      "not-implemented":"true",
+      "name": "Is string formatted in title case?"
+    },
+    "does_substring_exist": {
+      "not-implemented": "true",
+      "name": "Does a substring exist in a string?"
+    },
+    "find_start_index_of_substring": {
+      "not-implemented": "true",
+      "name": "Find index of where a substring starts"
+    },
+    "find_start_index_of_additional_substring": {
+      "not-implemented": "true",
+      "name": "Find index of an additional substring (or starting at another index)"
+    },
+    "find_start_index_of_substring_from_end": {
+      "not-implemented": "true",
+      "name": "Find substring index starting at end"
+    },
+    "count_occurrences_of_substring": {
+      "not-implemented": "true",
+      "name": "Find number of occurences of a substring"
+    },
+    "get_leftmost_characters": {
+      "not-implemented": "true",
+      "name": "Get a specified number of characters from the left"
+    },
+    "get_rightmost_characters": {
+      "not-implemented": "true",
+      "name": "Get a specified number of characters from the right"
+    },
+    "get_substring_from_start_and_end_index": {
+      "not-implemented": "true",
+      "name": "Return a substring from a string based on starting and ending indices"
+    },
+    "get_substring_from_start_index_and_length": {
+      "not-implemented": "true",
+      "name": "Return a substring from a string based on starting index and size of substring"
+    },
+    "convert_to_uppercase": {
+      "code": [
+        "#include<stdio.h>",
+        "#include<string.h>",
+        "char str[100];",
+        "gets(str);",
+        "printf(\"%s\",strupr(str);"
+      ],
```

Comment:
You can probably make this simpler and/or more direct by just using a pre-coded string. like:
```
#include<stdio.h>
#include<string.h>
char str[] = "Hello, world!";
printf(\"%s\",strupr(str);
```

Also at first I thought this was `isupper()` and you were wrong, but turns out we're both right! Maybe adding that in as a second option? (same for `islower()`)